packages/common/src/ui/components/navigation/Tabs/Tabs.tsx

```diff
@@ -19,7 +19,6 @@ export const TabList = styled(MuiTabList)({
 export const Tab = styled(MuiTab)({
   color: '#8f90a6',
   fontSize: '12px',
-  width: '100px',
```

Comment:
not sure what this will break 🤷 
The wider tabs in inbound shipments `Quantities (Ctrl+1)` were not fitting in!